plugin/etcd3/lookup_test.go

```diff
@@ -43,25 +42,6 @@ var services = []*msg.Service{
 	// Nameservers.
 	{Host: "10.0.0.2", Key: "a.ns.dns.skydns.test."},
 	{Host: "10.0.0.3", Key: "b.ns.dns.skydns.test."},
```

Comment:
why are these tests removed?

Reply:
thanks for the review, looks like there were few tests not running due to renaming. Now that I switched back to the plugin name `etcd` few tests are failing. I will try to fix these today hopefully